config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph.yaml

```diff
@@ -0,0 +1,137 @@
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-ceph
+spec:
+  deployStrategy:
+      deploy: false
+  services:
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+  baremetalSetTemplate:
+    deploymentSSHSecret: dataplane-ansible-ssh-private-key-secret
+    ctlplaneInterface: enp1s0
+    provisionServerName: openstackprovisionserver
+    ctlplaneGateway: 172.22.0.3
+    ctlplaneNetmask: 255.255.255.0
+    bmhNamespace: openstack
+    domainName: osptest.openstack.org
+  nodeTemplate:
+    nodes:
+      env:
+        - name: ANSIBLE_FORCE_COLOR
+          value: "True"
+        - name: ANSIBLE_ENABLE_TASK_DEBUGGER
+          value: "True"
+      edpm-compute-0:
+        hostName: edpm-compute-0
+        ansible:
+          ansibleHost: 192.168.122.100
+          ansibleVars:
+            ctlplane_ip: 192.168.122.100
+            internal_api_ip: 172.17.0.100
+            storage_ip: 172.18.0.100
+            tenant_ip: 172.19.0.100
+            fqdn_internal_api: edpm-compute-0.example.com
+          ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+        deployStrategy:
+          deploy: false
+      edpm-compute-1:
+        hostName: edpm-compute-1
+        ansible:
+          ansibleHost: 192.168.122.101
+          ansibleVars:
+            ctlplane_ip: 192.168.122.101
+            internal_api_ip: 172.17.0.101
+            storage_ip: 172.18.0.101
+            tenant_ip: 172.19.0.101
+            fqdn_internal_api: edpm-compute-1.example.com
+          ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+      preProvisioned: true
+    extraMounts:
```

Comment:
If `extraMounts` is to be removed, then this sample would need to be updated accordingly. Or I misunderstood @slagle's comment about removing it. Another option is to remove `extraMounts` in a follow up patch so as to limit the scope and get this merged sooner.

Reply:
I misunderstood, so please disregard this comment.